tox.ini

```diff
@@ -206,7 +206,7 @@ commands =
   lint-opentelemetry-sdk: isort --diff --check-only --settings-path {toxinidir}/.isort.cfg {toxinidir}/opentelemetry-sdk
   lint-opentelemetry-sdk: flake8 --config {toxinidir}/.flake8 {toxinidir}/opentelemetry-sdk
   lint-opentelemetry-sdk: pylint {toxinidir}/opentelemetry-sdk
-  benchmark-opentelemetry-sdk: pytest {toxinidir}/opentelemetry-sdk/benchmarks {posargs} --benchmark-json=sdk-benchmark.json
+  benchmark-opentelemetry-sdk: pytest {toxinidir}/opentelemetry-sdk/benchmarks --benchmark-json={toxinidir}/opentelemetry-sdk/sdk-benchmark.json {posargs}
```

Comment:
The problem is in the CI we are running `run: tox -e benchmark-opentelemetry-sdk -- -k opentelemetry-sdk/benchmarks --benchmark-json=opentelemetry-sdk/output.json` but the output.json is empty. with that I was able to get the output.json with the expected data.

Reply:
Another option is to keep the whole command in `tox.ini` and just call `tox -e benchmark-opentelemetry-sdk` on CI